source/initHQ/fortifyFOB.sqf

```diff
@@ -17,8 +22,8 @@ _aapod allowdamage false;
 _aapod setdamage 0;
 _aapod setpos [_centerpos select 0, _centerpos select 1,(_centerpos select 2)+3.109];
 
-_HQguard1 = _groupGuard createUnit ["B_Soldier_F", _centerPos, [], 0, "FORM"]; 
-_HQguard1 moveinGunner _aapod;
+_fobguard1 = _groupGuard createUnit ["B_Soldier_F", _centerPos, [], 0, "FORM"]; 
+_fobguard1 moveinGunner _aapod;
```

Comment:
Two questions here... They actually apply though the rest of the file.
1.) Why did we rename `_HQguard1`?
2.) If we are going to keep the variable named what it is could we please rename it in camel-case? Ex: `_fobGuard1`